soroban-react-dapp/package.json

```diff
@@ -8,7 +8,7 @@
     }
   },
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3001",
```

Comment:
but docker image is not sharing the 3001, need to add instructions in README

Reply:
this would work outside the docker

Reply:
The reason behind this was the container running on docker opens it up on port 3000, nextjs runs on 3000 by default also, i realized later frontend can run before exiting docker bash but at time when i was developing i made this adjustment for avoid a conflict. I understand this can be a potential breaking change and can impact the deployment on Vercel, However it's easy to rollback if it's necessary